services/users.js

```diff
@@ -767,6 +767,36 @@ module.exports = class UsersService {
     }, tokenOptions);
   }
 
+  static async verifyEmailConfirmationToken(token) {
```

Comment:
Difference to `verifyEmailConfirmation`?

Reply:
`verifyEmailConfirmation` actually marks the email as verified, and uses `verifyEmailConfirmationToken` under the hood. Comments that I'll add now should make that clear :)